lib/bump.rb

```diff
@@ -155,6 +157,21 @@ def bump(file, current, next_version, options)
         ["Bump version #{current} to #{next_version}", 0]
       end
 
+      def open_changelog options, log
+        if options[:editor]
+          return ["Could not find '#{options[:editor]}'", 1] unless command_exists?(options[:editor])
+          system(options[:editor], log)
+          puts "Ready with your Changelog? Y/n"
```

Comment:
if the editor exited successfully then we should be good to go, just like `git commit` works 🤷‍♂

Reply:
The editor can be sublime, vscode, gedit or something else that won't open in the terminal. That is why I want to wait for it.

Reply:
https://stackoverflow.com/questions/22631863/setting-up-sublimetext-3-as-git-commit-text-editor

if git does not need to support fanciness like that, then we don't either :trollface:

Reply:
... also again, best to use `ENV['EDITOR']` 🤷‍♂

Reply:
Okay I will look at how to use sublime from your example tomorrow.

What will happen if you have the flag `--editor`, but do not have `ENV['EDITOR']`

Reply:
I saw the sublime example. I do not know how to open the editor wait for it to complete...
Do you have a tip?

Reply:
`EDITOR="subl -n -w"` should do it ... if none is set, fall back to `vi`

Reply:
flag could be `--edit-changelog` which then also sets `changelog` to true

Reply:
`system("subl -n -w", "file")` does not work for me. Could you check it?

``nano file`` does not work eider..

I am thinking of adding an if to check for vi and nano in order to use ```` and `system()` for subl.